deploy/kubernetes/base/controller/cluster_setup.yaml

```diff
@@ -272,3 +272,12 @@ roleRef:
   kind: Role
   name: csi-gce-pd-leaderelection-role
   apiGroup: rbac.authorization.k8s.io
+
```

Comment:
Lets keep this in a seperate file? given that cluster setup has only rbac rules? @msau42  thoughts

Reply:
Done